### DISCUSSION TITLE
How do I set the default size for everything?

### DISCUSSION THREAD

**Discussion by tx46** [mantinedev/mantine#4034] [Q&A]
I want to be able to change the size of the entire UI. Unfortunately, setting up paddings and fontSizes does not seem to be enough. E.g., the paddings for TextInput will look all wrong if you do it that way. How can I configure the size of everything to be set to "xs" for example?

**Comment by rtivital** [comment]
`* { font-size: var(--mantine-font-size-xs) !important; }`

**Comment by disillusionedowl** [comment]
https://mantine.dev/theming/theme-object/

```js
export const theme = {
	colorScheme: 'light',
	globalStyles: theme => ( {
		html: {
			fontSize: '200%'
		}
	} )
};
```

Not exactly what you asked but there is that very hacky way to scale the UI. It will not work super well, but if you are desperate, you can try the above, perhaps complementing some other approaches. Maybe you will find some use for it. Mantine expects html fontSize to _always_ be at `100%` (so that 1rem equals to 16px), but you can technically change that to `50%`, `200%`, etc. Your mileage may vary, this approach is not officially supported.
